src/ZenohUTransport.cpp

```diff
@@ -315,11 +325,16 @@ v1::UStatus ZenohUTransport::sendPublishNotification_(
 	auto priority = mapZenohPriority(attributes.priority());
 
 	try {
+// suppresses the following: error: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Werror=pedantic]
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
@debruce - Could you try this instead? If it doesn't work, we can leave what you have here

```suggestion
#pragma GCC diagnostic ignored "-Wc++20-extensions"
```

Reply:
When I tried this, I get the following error:
`/home/david/projects/up-transport-zenoh-cpp/src/ZenohUTransport.cpp:280:32: error: unknown option after ‘#pragma GCC diagnostic’ kind [-Werror=pragmas]
  280 | #pragma GCC diagnostic ignored "-Wc++20-extensions"
      |                                ^~~~~~~~~~~~~~~~~~~~
`